pandas/tests/test_index.py

```diff
@@ -2517,6 +2518,18 @@ def test_string_categorical_index_repr(self):
                 self.assertEqual(unicode(idx), expected)
 
 
+class TestIntervalIndex(Base, tm.TestCase):
+    # see test_interval for more extensive tests
+    _holder = IntervalIndex
+
+    def setUp(self):
+        self.indices = dict(intvIndex = tm.makeIntervalIndex(100))
+        self.setup_indices()
+
+    def create_index(self):
+        return IntervalIndex.from_breaks(np.arange(0, 100, 10))
+
+
```

Comment:
I think you need a fair amount of additional testing methods. As ``Base`` only does so much. You need to do this like ``Categorical/MultiIndex`` (e.g. things like append/insert/delete)